### ISSUE TITLE
Queue System

### ISSUE THREAD

**Issue by henkhaus** [henkhaus/reddit_bot_testing#6]
create a queue system to capture palindromes in comments to reply to later.

**Comment by thagler1** [comment]
should we split the logic? We can have a palindrome finder constantly search redit for palindromes, when it finds one it will add it to the queue. Then have a second process that just replies to items in the queue.

**Comment by henkhaus** [comment]
yeah I am all about that. but I want to build this with a focus on
abstraction, using thepalindromebot as a working example. That way we can
implement other bots with the framework. Thoughts?

**Comment by thagler1** [comment]
k, That should be an epic of its own. And should probably be handled now. I think we need 3 components, a bot function, a reddit crawler, and a reddit poster. The bot function will be the only part that needs to be changed out in order to retool the bot. Once a bot function is defined, it will be passed to the crawler, that will then find posts for the reddit poster to reply to.

**Comment by henkhaus** [comment]
I agree. The bot class should hold all Reddit login, formatting, and
ancillary tracking that would be the same for all bots. The crawler finds
thing based one what logic that bot has defined and the poster will post as
it can.
